<launch>
  <arg name="INPUT_IMAGE"/>
  <arg name="model_file" default="$(find coral_usb)/models/seagrass128.tflite"/>
  <arg name="label_file" default="$(find coral_usb)/models/seagrass_labels.txt"/>

  <node name="edgetpu_patch_classifier"
        pkg="coral_usb" type="edgetpu_patch_classifier.py"
        output="screen" respawn="true">
    <remap from="~input" to="$(arg INPUT_IMAGE)" />
    <rosparam subst_value="true" >
      model_file: $(arg model_file)
      label_file: $(arg label_file)
    </rosparam>
  </node>
</launch>
